Src/Lecoati.LeBlender.Ui/App_Plugins/LeBlender/editors/leblendereditor/dialogs/parameterconfig.controller.js

```diff
@@ -277,26 +277,33 @@
 
     	$scope.isValid = function () {
 			var isValid = true;
-			
+
     		_.every($scope.model.value, function (item, itemIndex) {
     	         _.forEach(item, function (property, propertyIndex) {
     	            if (!property.$valid) {
 						isValid = false;
     					return;
     	            }
+					if (property.$editor.required && property.value == null) {
```

Comment:
The **property.value** before editing the input is empty string and not null. So to check right away you want to check for **!property.value** instead of **property.value == null**.